app/assets/stylesheets/profile.scss

```diff
@@ -1,29 +1,28 @@
 #profile_container {
   .profile_header {
     margin-bottom: 15px;
-    background-color: $framed-background;
-    padding-left: 10px;
-    padding-right: 10px;
-    padding-top: 20px;
-    margin-top: -20px;
-    box-shadow: $card-shadow;
+    padding: 5px;
+    margin-top: 10px;
+    text-shadow: 1px 1px 0px #fff;
```

Comment:
- Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.
- `0px` should be written without units as `0`